test/suite/test_verbose01.py

```diff
@@ -133,17 +139,31 @@ def test_verbose_none(self):
             # are generated.
             uri = 'table:test_verbose01_none'
             session = conn.open_session()
-            session.create(uri, 'key_format=S,value_format=S')
+            session.create(uri, self.collection_cfg)
             c = session.open_cursor(uri)
             c['none'] = 'none'
             c.close()
 
-    # Test use cases passing invalid verbose categories, ensuring the appropriate error message is raised.
+    # Test use cases passing invalid verbose categories, ensuring the appropriate error message is
+    # raised.
     def test_verbose_invalid(self):
         self.close_conn()
         self.assertRaisesHavingMessage(wiredtiger.WiredTigerError,
                 lambda:self.wiredtiger_open(self.home, 'verbose=[test_verbose_invalid]'),
-                '/\'test_verbose_invalid\' not a permitted choice/')
+                '/\'test_verbose_invalid\' not a permitted choice for key \'verbose\'/')
+
+    # Test use cases passing invalid verbose levels, ensuring the appropriate error message is
+    # raised.
+    def test_verbose_level_invalid(self):
```

Comment:
I think this test should live in `test_verbose02`, as `test_verbose01` is just about the legacy API and this is testing edge-cases in the new API.

Reply:
Oh wait, this can be removed as its already in `test_verbose02`

Reply:
Thanks, I missed that one! Removed in 357e2c72997a72c6bff6acce6471aa60b04039a8.